perf/istio-install/base/templates/prometheus-install.yaml

```diff
@@ -16,6 +16,15 @@ metadata:
   name: prometheus
   namespace: istio-prometheus
 spec:
+  alerting:
+    alertmanagers:
+      - name: alertmanager-main
+        namespace: istio-prometheus
+        port: web
+  ruleSelector:
+    matchLabels:
+      role: prometheus-example-rules
+      prometheus: example
   image: "docker.io/prom/prometheus:v2.17.2"
```

Comment:
while we are at it, we may want to bump to 2.19. it supposedly has much improved memory usage.

Reply:
ok done